### PR TITLE
Fixed test again

### PR DESCRIPTION
**Pull request by david-puglielli**
Apparently the error code on Bamboo is different from the one tested on my mac.

---
This change is [Reviewable](https://reviewable.io/reviews/microsoft/msphpsql/741)

### CODE REVIEW

File: test/functional/pdo_sqlsrv/pdo_utf8_conn.phpt [hunk 1]
```diff
@@ -21,7 +21,7 @@ if ($c !== false) {
 ?>
 --EXPECTREGEX--
 
-Fatal error: Uncaught PDOException: SQLSTATE\[(28000|08001|HYT00)\]: .*\[Microsoft\]\[ODBC Driver 1[0-9] for SQL Server\](\[SQL Server\])?(Named Pipes Provider: Could not open a connection to SQL Server \[2\]\. |TCP Provider: Error code 0x2AF9|Login timeout expired|Login failed for user 'sa'\.) in .+(\/|\\)pdo_utf8_conn\.php:[0-9]+
+Fatal error: Uncaught PDOException: SQLSTATE\[(28000|08001|HYT00)\]: .*\[Microsoft\]\[ODBC Driver 1[0-9] for SQL Server\](\[SQL Server\])?(Named Pipes Provider: Could not open a connection to SQL Server \[2\]\. |TCP Provider: Error code 0x2726|TCP Provider: Error code 0x2AF9|Login timeout expired|Login failed for user 'sa'\.) in .+(\/|\\)pdo_utf8_conn\.php:[0-9]+
 Stack trace:
```
Comment (yitam) [inline test/functional/pdo_sqlsrv/pdo_utf8_conn.phpt]: can you change the regex to `TCP Provider: Error code` to accept either one of the two possible codes?

Comment (david-puglielli) [reply test/functional/pdo_sqlsrv/pdo_utf8_conn.phpt]: That's `TCP Provider: Error code (0x2726|0x2AF9)` right?

Comment (yitam) [reply test/functional/pdo_sqlsrv/pdo_utf8_conn.phpt]: should be, please run a quick test in your local env just in case :)

**Comment by david-puglielli** [comment]
Merging because I want to run the fix tonight.

**Pull request merged**